docker-compose.yml

```diff
@@ -31,6 +39,7 @@ services:
       - "env_files/back-secret.env"
     environment:
       BUNDLE_PATH: /bundle
+      REDIS_URL: redis://redis:6379
```

Comment:
Adding `REDIS_URL` env value stopped getting this redis connection error:
```Ruby
[3] pry(main)> Redis.new.pubsub("channels", "action_cable/*")
Redis::CannotConnectError: Error connecting to Redis on 127.0.0.1:6379 (Errno::ECONNREFUSED)
from /bundle/ruby/2.7.0/gems/redis-3.3.5/lib/redis/client.rb:345:in `rescue in establish_connection'
Caused by Errno::ECONNREFUSED: Connection refused - connect(2) for 127.0.0.1:6379
from /usr/local/lib/ruby/2.7.0/socket.rb:1214:in `__connect_nonblock'
Caused by IO::EINPROGRESSWaitWritable: Operation now in progress - connect(2) would block
from /usr/local/lib/ruby/2.7.0/socket.rb:1214:in `__connect_nonblock'
```
With this env var specified, we now get...
```Ruby
pry(main)> Redis.new.pubsub("channels", "action_cable/*")
=> []
```